app/services/doc_auth/lexis_nexis/config.rb

```diff
@@ -5,8 +5,6 @@ module LexisNexis
       :base_url, # required
       :request_mode,
       :trueid_account_id,
-      :trueid_liveness_cropping_workflow,
-      :trueid_liveness_nocropping_workflow,
       :trueid_noliveness_cropping_workflow,
       :trueid_noliveness_nocropping_workflow,
```

Comment:
Can we simplify this to avoid references to liveness altogether?

```suggestion
      :trueid_cropping_workflow,
      :trueid_nocropping_workflow,
```

Reply:
I think we could, but either the new config names will need to be promoted through all evns or we will need to have the name true id uses be out of sync.